src/SFA.DAS.EmployerIncentives.Web/Controllers/ApplyApprenticeshipsController.cs

```diff
@@ -94,11 +104,24 @@ public async Task<IActionResult> ConfirmApprenticeships(string accountId, Guid a
 
         [HttpPost]
         [Route("confirm-apprentices/{applicationId}")]
-        public async Task<IActionResult> DisplayDeclaration(string accountId, Guid applicationId)
+        public async Task<IActionResult> DisplayDeclaration(string accountId, Guid applicationId, bool newAgreementRequired)
         {
+            if(newAgreementRequired)
```

Comment:
We are relying on the posted data here.  If the newAgreementRequired is spoofed is the agreement version checked during the apply process?

Reply:
I'd considered this and was comfortable with the fact they'd be blocked by the new validation rule at point of payment.

Reply:
OK.  Is it worth adding a comment in the code so it's been considered?  In case someone uses it as reference code and codes a loop hole.